test/units/testsuite-65.sh

```diff
@@ -51,6 +51,21 @@ systemd-analyze dot --require systemd-journald.service systemd-logind.service >/
 systemd-analyze dot "systemd-*.service" >/dev/null
 (! systemd-analyze dot systemd-journald.service systemd-logind.service "*" bbb ccc)
 # dump
+# this should be rate limited to 10 calls in 10 minutes for unprivileged callers
```

Comment:
We don't have the `test/units/util.sh` script in RHEL9, so you'll need to copy the definition of `runas()` here:

https://github.com/systemd/systemd/blob/e3828d7103a99a15a1e947ba3063294ead590631/test/units/util.sh#L133C1-L137C2

Reply:
addressed in https://github.com/redhat-plumbers/systemd-rhel9/pull/267/commits/27e836bd3eacb25d0a714cf59b2272316670ae26